search-app/src/cmr/search/services/query_execution/has_granules_or_cwic_results_feature.clj

```diff
@@ -38,9 +46,32 @@
           (for [coll-id (map :_id (get-in results [:hits :hits]))]
             [coll-id 1]))))
 
+(defn get-opensearch-collections
+  "Returns the collection granule count by searching elasticsearch by aggregation"
+  [context provider-ids]
+  (let [condition (gc/or-conds [(qm/string-conditions :consortiums ["CWIC"])
+                                (qm/string-conditions :consortiums ["FEDEO"])
+                                (qm/string-conditions :consortiums ["GEOSS"])
+                                (qm/string-conditions :consortiums ["CEOS"])
+                                (qm/string-conditions :consortiums ["EOSDIS"])])
```

Comment:
Possibly add a `defconfig` someplace with a default list of consortiums instead of having to list them out individually all the time